src/molecules/ComingSoonPage/index.test.tsx

```diff
@@ -0,0 +1,39 @@
+import { render, screen} from '@testing-library/react';
+import ComingSoonPage from './index';
+import config from './config.json';
+
+describe('Coming Soon component', () => {
+  test('renders without crashing', () => {
+    render(<ComingSoonPage />);
+  });
+
+  test('displays "Coming Soon Title" text', () => {
+    const { getByText } = render(<ComingSoonPage />);
+    const textElement = getByText(config.component.title);
+    expect(textElement).toBeInTheDocument();
+  });
+
+  test('displays "Description" text', () => {
+    const { getByText } = render(<ComingSoonPage />);
+    const textElement = getByText(config.component.description);
+    expect(textElement).toBeInTheDocument();
+  });
+
```

Comment:
there are no fallback conditions like what happen if no value is passed to config

Reply:
Have merged with base-setup, used customThemeHook for changing theme, added fallback condition
![image](https://github.com/SamagraX-Stencil/ui-templates/assets/161353600/c10460eb-0c86-43f1-a057-2b9e73c15bd3)